dandi/tests/fixtures.py

```diff
@@ -312,3 +320,53 @@ def upload_dandiset(paths=None, **kwargs):
         "dandiset_id": dandiset_id,
         "reupload": upload_dandiset,
     }
+
+
+@pytest.fixture(scope="session")
+def download_video_files():
+    from datalad.api import Dataset, install
+    pt = Path(tempfile.mkdtemp())/"ophys_testing_data"
```

Comment:
Use pytest's [`tmp_path_factory`](https://docs.pytest.org/en/6.2.x/reference.html#tmp-path-factory) fixture instead of `tempfile`.